policy/modules/services/bird.fc

```diff
@@ -11,3 +11,5 @@
 /var/log/bird\.log.*	--	gen_context(system_u:object_r:bird_log_t,s0)
 
 /run/bird\.ctl	-s	gen_context(system_u:object_r:bird_runtime_t,s0)
+
+/run/bird	-d	gen_context(system_u:object_r:bird_runtime_t,s0)
```

Comment:
Why not have the directory's contents too?

Reply:
Fixed.